ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -22,38 +22,46 @@ class DrawerPresentationController: UIPresentationController {
+	init(presentedViewController: UIViewController, presenting presentingViewController: UIViewController?, source: UIViewController, sourceObject: Any?, presentationOrigin: CGFloat?, presentationDirection: DrawerPresentationDirection, presentationOffset: CGFloat, presentationBackground: DrawerPresentationBackground, adjustHeightForKeyboard: Bool, shouldUseWindowFullWidthInLandscape: Bool, passThroughView: UIView?) {
         sourceViewController = source
         self.sourceObject = sourceObject
         self.presentationOrigin = presentationOrigin
         self.presentationDirection = presentationDirection
         self.presentationOffset = presentationOffset
         self.presentationBackground = presentationBackground
         self.shouldUseWindowFullWidthInLandscape = shouldUseWindowFullWidthInLandscape
+        self.passThroughView = passThroughView
         super.init(presentedViewController: presentedViewController, presenting: presentingViewController)
 
-        backgroundView.gestureRecognizers = [UITapGestureRecognizer(target: self, action: #selector(handleBackgroundViewTapped(_:)))]
-
         if adjustHeightForKeyboard {
             NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardWillChangeFrame), name: UIApplication.keyboardWillChangeFrameNotification, object: nil)
         }
     }
 
-    // A transparent view, which if tapped will dismiss the dropdown
     private lazy var backgroundView: UIView = {
+    // A transparent view, which if tapped will dismiss the dropdown
         let view = BackgroundView()
+        view.forwardsTouches = false
+        // Pass the passthrough view in touch forwarding view
+        if let passThroughView = self.passThroughView {
+            view.passthroughView = passThroughView
+        }
         view.backgroundColor = .clear
         view.isAccessibilityElement = true
         view.accessibilityLabel = "Accessibility.Dismiss.Label".localized
         view.accessibilityHint = "Accessibility.Dismiss.Hint".localized
         view.accessibilityTraits = .button
         // Workaround for a bug in iOS: if the resizing handle happens to be in the middle of the backgroundView, VoiceOver will send touch event to it (according to the backgroundView's accessibilityActivationPoint) even though it's not parented in backgroundView or even interactable - this will prevent backgroundView from receiving touch and dismissing controller
+
+        view.gestureRecognizers = [UITapGestureRecognizer(target: self, action: #selector(handleBackgroundViewTapped(_:)))]
         view.onAccessibilityActivate = { [unowned self] in
             self.presentingViewController.dismiss(animated: true)
         }
         return view
     }()
+
     private lazy var dimmingView: DimmingView = {
```

Comment:
any we need to change in the dimmingView bound to make sure the rounded corners of the drawerControlller's view is shown?

Reply:
I didn't change the dimming view. Its bounds are already changing. Please let me know if you meant something else.

Reply:
![Screen Shot 2020-08-31 at 10 30 30 AM](https://user-images.githubusercontent.com/20715435/91749168-f0179600-eb75-11ea-862f-08decb3986fa.png)
There isn't much shadow around the drawer that I can't see the rounded corner or the difference between the passthrough view and the drawercontroller view.

Reply:
There is DrawerShadowView which is being used in the drawer for shadow. Is that the right place to do the changes for shadow?

Reply:
I checked, DrawerShadowView is supposed to provide the shadow to the bottom sheet. Looks like adding below line in one of the commit (4cf4800) break the shadow view.
**contentView.clipsToBounds = true** 

**4cf4800**: add a white mask to MSDrawerPresentationController so that during presentation it won't slide over the navigation bar.

Right now shadowView is the part of contentview (where we are setting clipToBounds to be true). What we can do is to add shadowView in the container view. It fixes the issue. Thoughts?

Reply:
Fixed!